lib/modules/datasource/adoptium-java/readme.md

```diff
@@ -1,10 +1,10 @@
-This datasource returns releases from [Adoptium](https://adoptium.net/) API.
+This datasource returns releases from the [Adoptium](https://adoptium.net/) API.
 
 It uses `image_type=<jre|jdk>&project=jdk&release_type=ga&sort_method=DATE&sort_order=DESC&vendor=adoptium` as filter parameters.
```

Comment:
I'm not sure what `filter parameters` means here. The `image_type=<jre|jdk>&project=jdk&release_type=ga&sort_method=DATE&sort_order=DESC&vendor=adoptium` is a bit tricky to parse on first glance.

How about we add a plain language explanation like this maybe?

The datasource will check:

- JRE or JDK files
- with a JDK project
- which have the GA (general availability) status
- and sorts the results in descending order
- only checks vendor type Adoptium

Reply:
sure, vendor filter has been removed. I Don't really know the details of the `project` filter. I use mainly the default values

https://api.adoptium.net/q/swagger-ui/#/Release%20Info/getReleaseVersions